catalog/entry_catalog.go

```diff
@@ -129,7 +134,10 @@ func NewEntryCatalog(cfg *config.Config, db db.Database) (*EntryCatalog, error)
 	refManager := ref.NewPGRefManager(db, ident.NewHexAddressProvider())
 	branchLocker := ref.NewBranchLocker(db)
 	store := graveler.NewGraveler(branchLocker, committedManager, stagingManager, refManager)
-	entryCatalog := &EntryCatalog{Store: store}
+	entryCatalog := &EntryCatalog{
+		BlockAdapter: tierFSParams.Adapter,
```

Comment:
TierFSParams??

Reply:
Result of GetCommittedTierFSParams includes the configured block adapter used by the storage.